src/js/modules/i18n/regional/infragistics.ui.regional-af.js

```diff
@@ -51,8 +51,9 @@
 	    currencyPositivePattern: '$ n',
 	    currencyNegativePattern: '$-n',
 	    currencySymbol: 'R'
-    };
-    if ($.ig.setRegionalDefault) {
-	    $.ig.setRegionalDefault('af');
+	};
+	
+    if ($.ig.util.changeGlobalRegional) {
```

Comment:
The old check for `$.ig.setRegionalDefault` was safe due to the code above ensuring the `$.ig` object.
This will throw `..'changeGlobalRegional' of undefined` if the regional files end up before the util.
Valid for all regional files.